File: output/ChatGPT/codenet/Python/Java/s013976107.java
int A = scanner.nextInt();
int B = scanner.nextInt();
String S = scanner.next();
if (S.charAt(A) == '-' && S.chars().filter(ch -> ch == '-').count() == 1) {
    System.out.println("Yes");
} else {
    System.out.println("No");
}

File: output/ChatGPT/codenet/Python/C++/s013976107.cpp
#include <iostream>
#include <vector>
using namespace std;

int main() {
    int A, B;
    cin >> A >> B;
    string S;
    cin >> S;
    
    if (S[A] == '-' && count(S.begin(), S.end(), '-') == 1) {
        cout << "Yes" << endl;
    } else {
        cout << "No" << endl;
    }
    
    return 0;
}